### ISSUE TITLE
Refugee Center has nether chewed field terrain

### ISSUE THREAD

**Issue by danielkwinsor** [CleverRaven/Cataclysm-DDA#66233]
### Describe the bug

Reach Refugee Center mission existed.  When portal storm came, had the refugee center become nether chewed field, while the nearby terrain stayed normal.

### Attach save file

[Central Lake - 9Portal Storm-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/11765840/Central.Lake.-.9Portal.Storm-trimmed.tar.gz)

### Steps to reproduce

Find evac shelter
Get coordinates
Wait for portal storm

### Expected behavior

Nether-chewed field is around the portal, not around refugee center

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.3086 (22H2)
- Game Version: 020c81a [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Alternative Map Key [alt_map_key]
]

**Comment by danielkwinsor** [comment]
This doesn't appear to be related to portal storm, I just thought it was due to "nether" and to time proximity.  This seems to be some change in a recent experimental version about the terrain.

**Comment by danielkwinsor** [comment]
06-14-1544 is ok
06-14-2329 is bad
change introduced sometime after 14-1544

Didn't track it down any further... it would be great to have changelog in experimental builds.

**Comment by Procyonae** [comment]
I'm presuming I'm responsible for this #66123 but I have no idea how portal storms work as to why this happened, I'll have a dig >_<

**Comment by Procyonae** [comment]
Yep I totally forgot to migrate the old OMTs bc I'm stoopid, this isn't related to portal storms in any way though as far as I can tell the nether chewed OMT is just the fallback (like the abandoned drive thrus back along)

**Comment by SurFlurer** [comment]
This is not fully fixed by #66235. I still get error messages like :
```
 DEBUG    : Loaded invalid oter_id 'refctr_NE5a_z-4_west'

 FUNCTION : unserialize
 FILE     : E:\Cataclysm-DDA\src\savegame.cpp
 LINE     : 442
 VERSION  : 0.G-2380-ga272beb2b3

```

**Comment by Procyonae** [comment]
Somehow I didn't notice I used only the north variant and my test world happened to spawn a north facing variant :facepalm:
